java/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version:  '3.2'
```

Comment:
Why a Docker file?  We don't use Docker for anything else in the language examples; and it's a little embarrassing to have in there given that we don't offer Docker to customers. :smile:

Reply:
Removed